x-pack/plugins/apm/public/constants/index.ts

```diff
@@ -4,8 +4,17 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { i18n } from '@kbn/i18n';
+
 export const STATUS = {
   FAILURE: 'FAILURE',
   SUCCESS: 'SUCCESS',
   LOADING: 'LOADING'
 };
+
+export const NOT_AVAILABLE_LABEL = i18n.translate(
+  'xpack.apm.notAvailableLabel',
+  {
+    defaultMessage: 'N/A'
+  }
+);
```

Comment:
Thanks for doing this! Can you also replace the remaining N/A labels?
https://github.com/elastic/kibana/search?q=notAvailableLabel+path%3Ax-pack%2Fplugins%2Fapm%2F&unscoped_q=notAvailableLabel+path%3Ax-pack%2Fplugins%2Fapm%2F

If it's outside the scope of your work LMK and I'll see to it.

Reply:
I did it here #28677

Reply:
Oh,, that's right. Awesome!